### ISSUE TITLE
unknown font id 0 (when including aar that has custom fonts in /raw)

### ISSUE THREAD

**Issue by loeschg** [robolectric/robolectric#1426]
I'm using the droidicon library in my application (https://github.com/theDazzler/droidicon), and Robolectric tests are failing when I use it in a layout. 

I've included the library in my project.properties file, and it still fails. 

    android.library.reference.3=../../build/intermediates/exploded-aar/com.github.theDazzler/droidicon/0.1.6

Is there another line or some other configuration to make this work? Looking at the exploded-aar, there are multiple fonts included in the res/raw directory in case that's of relevance.

```
Caused by: java.lang.RuntimeException: unknown font id 0
	at org.robolectric.shadows.ShadowTypeface.findById(ShadowTypeface.java:105)
	at org.robolectric.shadows.ShadowTypeface.__constructor__(ShadowTypeface.java:32)
	at android.graphics.Typeface.<init>(Typeface.java:173)
	at android.graphics.Typeface.createFromFile(Typeface.java:169)
	at com.thedazzler.droidicon.util.TypefaceManager.createTypefaceFromResource(TypefaceManager.java:85)
	at com.thedazzler.droidicon.util.TypefaceManager.access$000(TypefaceManager.java:20)
	at com.thedazzler.droidicon.util.TypefaceManager$IconicTypeface.getTypeface(TypefaceManager.java:56)
	at com.thedazzler.droidicon.IconicFontDrawable.updateIcon(IconicFontDrawable.java:220)
	at com.thedazzler.droidicon.IconicFontDrawable.setIcon(IconicFontDrawable.java:74)
	at com.thedazzler.droidicon.badges.DroidiconBadge.initialise(DroidiconBadge.java:146)
	at com.thedazzler.droidicon.badges.DroidiconBadge.<init>(DroidiconBadge.java:62)
```

**Comment by freezy** [comment]
Same issue here:

```
java.lang.RuntimeException: unknown font id 0
	at org.robolectric.shadows.ShadowTypeface.findById(ShadowTypeface.java:105)
	at org.robolectric.shadows.ShadowTypeface.__constructor__(ShadowTypeface.java:32)
	at android.graphics.Typeface.<init>(Typeface.java:173)
	at android.graphics.Typeface.createFromFile(Typeface.java:159)
	at com.malinskiy.materialicons.Iconify.getTypeface(Iconify.java:67)
	at com.malinskiy.materialicons.Iconify.addIcons(Iconify.java:33)
	at com.malinskiy.materialicons.widget.IconTextView.init(IconTextView.java:38)
	at com.malinskiy.materialicons.widget.IconTextView.<init>(IconTextView.java:15)
	...
```

This is a blocker. Any suggestions?

**Comment by erd** [comment]
This should be fixed by d7c87e4a951f2e1e38cc65a2af7ddf31844abd4c.
